zwave/Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN \
         pwgen \
         qt5-qtbase \
         qt5-qtremoteobjects \
+        qt5-qtserialport \
```

Comment:
What does this change mean?

Reply:
It's a dependency that was added to qt-openzwave, required for it to compile.